_posts/2020/2020-09-30-design-matters-2020-conference.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Design Matters 2020
+date: 2020-09-30
+header_image: public/design-matters-2020.jpg
+header_overlay: true
+category: event
+tags: ["conference", "design"]
+authors: ["Sarah"]
+about_authors: ["swilsonreissmann"]
+---
+
+Just when you thought you wouldn’t be able to attend any conferences or do any networking in this year of COVID, the [Design Matters](https://designmatters.io/){:target="_blank"} conference in Copenhagen decided to go hybrid.
+Disappointed to not be able to attend in person, last week I eagerly logged on remotely for the two-day conference from the comfort of my own home.
+Design Matters is a yearly conference bringing together “creatives” who are interested in exploring new movements in digital design.
+From UX designers to product designers to UI designers, anyone who “creates” digital products will find something to be inspired by.
+
+## A hybrid conference 
+
+Using the live-streaming platform [vi.to](https://vi.to/){:target="_blank"} and guided by live stream host [André Cardoso](hhttps://designmatters.io/line-up/andre-cardoso/){:target="_blank"}, those of us participating remotely enjoyed main stage talks, online-only workshops, and interviews with speakers.
```

Comment:
hhttps://designmatters.io/line-up/andre-cardoso/

--> The link currently doesn't work because you start with hhttps instead of https